examples/performance/bracket/bracket.cxx

```diff
@@ -109,7 +110,7 @@ int main(int argc, char **argv) {
       time_output << "Case legend";
       time_output << "\n------------------------------------------------\n";
 
-      for (int i = 0; i < names.size(); i++) {
+      for (uint i = 0; i < names.size(); i++) {
```

Comment:
`uint` is non-standard. `std::size_t`, or a type alias for `std::vector<std::string>::size_type` would be more appropriate